src/view/treeNodes/pullRequestNode.ts

```diff
@@ -11,14 +11,13 @@ import { SlimFileChange, GitChangeType } from '../../common/file';
 import Logger from '../../common/logger';
 import { Resource } from '../../common/resources';
 import { fromPRUri, toPRUri } from '../../common/uri';
-import { groupBy, formatError } from '../../common/utils';
+import { groupBy, formatError, IDisposable } from '../../common/utils';
```

Comment:
We can probably consider replacing `IDisposable` with `vscode.Disposable`, I can't remember why I added this interface at the very beginning.